From 738ccfd93b7bf88e06795a962777f9dd82d05d10 Mon Sep 17 00:00:00 2001
From: Wei-Chiu Chuang <weichiu@cloudera.com>
Date: Mon, 2 May 2016 06:35:49 -0700
Subject: [PATCH 1480/2848] HADOOP-12789. log classpath of
 ApplicationClassLoader at INFO level. (Sangjin
 Lee via mingma)

(cherry picked from commit 49eedc7ff02ea61764f416f0e2ddf81370aec5fb)
(cherry picked from commit f01f1940c4006eeffae679f822463d156c790c04)

Change-Id: I9da281a5a0655b17dc09620aad208b203dd5d9e4
---
 .../apache/hadoop/util/ApplicationClassLoader.java |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/ApplicationClassLoader.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/ApplicationClassLoader.java
index b18997c..bec88bc 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/ApplicationClassLoader.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/ApplicationClassLoader.java
@@ -96,10 +96,6 @@ public boolean accept(File dir, String name) {
   public ApplicationClassLoader(URL[] urls, ClassLoader parent,
       List<String> systemClasses) {
     super(urls, parent);
-    if (LOG.isDebugEnabled()) {
-      LOG.debug("urls: " + Arrays.toString(urls));
-      LOG.debug("system classes: " + systemClasses);
-    }
     this.parent = parent;
     if (parent == null) {
       throw new IllegalArgumentException("No parent classloader!");
@@ -108,6 +104,7 @@ public ApplicationClassLoader(URL[] urls, ClassLoader parent,
     this.systemClasses = (systemClasses == null || systemClasses.isEmpty()) ?
         Arrays.asList(StringUtils.getTrimmedStrings(SYSTEM_CLASSES_DEFAULT)) :
         systemClasses;
+    LOG.info("classpath: " + Arrays.toString(urls));
     LOG.info("system classes: " + this.systemClasses);
   }
 
-- 
1.7.9.5

